pkg/gofr/container/container.go

```diff
@@ -73,8 +73,13 @@ func (c *Container) Create(conf config.Config) {
 	}
 
 	if c.Logger == nil {
+		levelFetchConfig, err := strconv.Atoi(conf.GetOrDefault("REMOTE_LOG_FETCH_INTERVAL", "15"))
+		if err != nil {
+			levelFetchConfig = 15
+		}
+
 		c.Logger = remotelogger.New(logging.GetLevelFromString(conf.Get("LOG_LEVEL")), conf.Get("REMOTE_LOG_URL"),
-			conf.GetOrDefault("REMOTE_LOG_FETCH_INTERVAL", "15"))
+			time.Duration(levelFetchConfig)*time.Second)
```

Comment:
```suggestion
			levelFetchConfig*time.Second)
```

Reply:
@ccoVeille  I didn't understand why are you eliminating these lines for fetching config values:
`conf.GetOrDefault("REMOTE_LOG_FETCH_INTERVAL", "15")`

Where and how will we fetch the config values then?

Reply:
I had only removed the type conversion, not the value

But I was wrong, the only way to multiply a duration is to case the type ad you did.

![Screenshot_20240807_133617_Firefox.jpg](https://github.com/user-attachments/assets/db02cdc7-b464-4269-88fd-6772fed24201)

So your code is fine

Reply:
thankyou